src/test/java/umk/mat/pajda/ProjektZespolowy/Services/ReviewServiceIntTestOfHashRevID.java

```diff
@@ -0,0 +1,118 @@
+package umk.mat.pajda.ProjektZespolowy.Services;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.bcrypt.BCrypt;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+import umk.mat.pajda.ProjektZespolowy.DTO.ReviewPatchPostDTO;
+import umk.mat.pajda.ProjektZespolowy.entity.Review;
+import umk.mat.pajda.ProjektZespolowy.entity.User;
+import umk.mat.pajda.ProjektZespolowy.misc.ReviewConverter;
+import umk.mat.pajda.ProjektZespolowy.repository.ReviewRepository;
+import umk.mat.pajda.ProjektZespolowy.repository.UserRepository;
+import umk.mat.pajda.ProjektZespolowy.services.ReviewService;
+
+@SpringBootTest
+@ActiveProfiles("tests")
+public class ReviewServiceIntTestOfHashRevID {
+
+  @Value("${FIXEDSALT_IPHASH:$2a$10$9elrbM0La5ooQgMP7i9yjO}")
```

Comment:
This test never reads the value from the FIXEDSALT_IPHASH variable, it always takes that default value.

Reply:
ok.